css/style.css

```diff
@@ -1,4 +1,38 @@
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300,400,400italic,700);
+/*
+ * Muli is licensed under SIL OFL Version 1.1
+ * See /assets/logos/font/Muli/OFL.txt file for full information.
+ */
+@font-face {
+    font-family: 'Muli';
+    src: url('/assets/logos/font/Muli/Muli-Bold.woff') format('woff');
```

Comment:
Where did you get the woff files from? I couldn't find them on Google Fonts.
Also, do we also want to serve woff2 versions if you have them as they're usually smaller?

Reply:
Google serves files depending on the user agent. I'll see if it's worth it adding woff2